### DISCUSSION TITLE
Inaccessibility of sticky toolbars

### DISCUSSION THREAD

**Discussion by Zalathar** [community/community#55805] [Accessibility]
### Select Topic Area

Product Feedback

### Body

Over the last several years, GitHub has been adding more and more sucky toolbars to its pages, seemingly unaware of the accessibility problems that they cause.

Most recently, the overhaul of the code view has added a mandatory sticky toolbar that remains at the top of the browser view even when scrolling.

Of course, this makes it very difficult for me to read the code on the page. I read a lot of code on GitHub, but since this update I’ve been unable to get any work done, because the sticky toolbar prevents me from being able to read the code on the page.

Is this something that GitHub’s accessibility people are even aware of? Frontend changes have been steadily making the site less and less accessible, and this isn’t consistent with GitHub’s claim that it cares about accessibility.

**Comment by cehfisher** [comment]
Thank you for your feedback @Zalathar - we genuinely appreciate your input. Our team has reviewed your request and we want to be transparent about our priorities. This particular request is not currently being actively worked on. This doesn’t imply that we’re not considering it in the future, but for now, it is not on our development roadmap.

We encourage you to check our [public roadmap](https://github.com/github/roadmap/projects/1) to get insights into the projects we are currently focusing on for the upcoming year. Please note that smaller fixes may not always be listed explicitly but can often be addressed when related work is in progress.

As a valuable GitHub user, your feedback in this Discussion space, accompanied by appropriate tags and examples, plays a crucial role in shaping GitHub’s development. While we can’t guarantee immediate action on every piece of feedback, rest assured that we are attentive to your suggestions and always striving to enhance the GitHub experience.

Once again, thank you for taking the time to share your thoughts with us, even if the particular request is not currently in active development. Your input is valuable and contributes to the continuous improvement of GitHub!
